lib/parse.js

```
@@ -217,6 +218,7 @@ var normalizeParseOptions = function normalizeParseOptions(opts) {
     var charset = typeof opts.charset === 'undefined' ? defaults.charset : opts.charset;
 
     return {
+        allowEmptyArrays: typeof opts.allowEmptyArrays === 'boolean' ? opts.allowEmptyArrays : defaults.allowEmptyArrays,
```

let's also throw if it's not a boolean or undefined